### ISSUE TITLE
Incorrect display of long room\recordings names!

### ISSUE THREAD

**Issue by nikkei00** [bigbluebutton/greenlight#1372]
I found this problem when creating rooms with very long names.
The text of the name is not transferred, and because of this, on the page "Server Rooms and Server Records", the control buttons and other information go to the right!

**I found a fix, executed it in this local file on the server, but after restarting everything remained as before.**

How can I solve this problem?

![0116](https://user-images.githubusercontent.com/64165027/80026898-edd06b80-84f3-11ea-8f82-773c3f3a0896.png)

![0117](https://user-images.githubusercontent.com/64165027/80026901-ef019880-84f3-11ea-9dfa-de7e20ebccca.png)

**Comment by ChristianGaetcke** [comment]
Hi, this can be fixed by a change in https://github.com/bigbluebutton/greenlight/blob/2af1f7242fe8993acc3b61ccccc1a54b6b6c9436/app/views/admins/components/_rooms.html.erb#L19

and removing the `text-nowrap`. But it will remove the house symbol and I don't know if it has any other impacts.

Best

**Comment by nikkei00** [comment]
I don't understand where exactly it needs to be fixed to make it work for me. The change should be made not on the local server, but in the repository and it should be downloaded and installed? I don't know much about it yet.

**Comment by ChristianGaetcke** [comment]
Hi,

This would be the starting point for Greenlight customization: https://docs.bigbluebutton.org/greenlight/gl-customize.html#customizing-greenlight

In general it would mean: You fork the github repo, clone it to your computer and do your changes and then create a new docker-container. This one will then be used.

If I were super confident, I'd create a PullRequest, but as I don't know the full impact of this change, I won't do it (for now)

**Comment by farhatahmad** [comment]
Removing that class would make that table unusable on mobile. The column can be given a max width to help make it look better

**Comment by nikkei00** [comment]
> Removing that class would make that table unusable on mobile. The column can be given a max width to help make it look better

Please tell me, where in the code will it be correct to set the column width? In percentage terms, it would be more correct, because the monitors are all different, right? I will try to copy the project and make changes according to the message above.

**Comment by mrkeksi** [comment]
https://github.com/bigbluebutton/greenlight/blob/master/app/assets/stylesheets/admins.scss

```
#room-name-text {
    line-height: 1.5em;
    max-height: 3em;
    overflow: hidden;
    max-width: 310px;
    text-overflow: ellipsis;
    display: -webkit-box;
    -webkit-line-clamp: 2;
    -webkit-box-orient: vertical;
    white-space: normal;
}
```

**Comment by nikkei00** [comment]
The problem is solved. Went to the Docker container where greenlight is running, there in the page layout from the <table > line ............> with server records and rooms, I deleted the text-nowrap class. saved the file, reloaded the Docker - everything is super.

**Comment by nikkei00** [comment]
Complete steps for users of my level who don't understand much:
docker ps -a / list all containers
docker exec -it [container-id] bash / enter in greenlight container

vi /usr/src/app/app/views/admins/components/_recordings.html.erb / open file
delete text-wrap in title table
:wq

vi /usr/src/app/app/views/admins/components/_rooms.html.erb / open file
delete text-wrap in title table
:wq

docker restart [container-id]

profit

**Comment by mrkeksi** [comment]
Added a fix in #1403
